src/views/ManageList.jsx

```diff
@@ -103,6 +120,21 @@ export function ManageList() {
 				<br />
 				<button type="submit">Submit</button>
 			</form>
+
+			{/* invite a user to share list form */}
+			<form onSubmit={handleEmailInputSubmit}>
+				<label htmlFor="email-input">Enter Email:</label>
+				<input
+					type="text"
```

Comment:
Since we're capturing an email address here, you could leverage the `email` type to improve form validation by using the browser's native email validation!

Reply:
thank you, fixed!